src/components/crownConfig.vue

```diff
@@ -0,0 +1,275 @@
+<template>
+  <div class="crown-config" :style="style" v-if="showCrown">
+    <sequence-flow-button
+      @click="addSequence()"
+      class="crown-config__icon"
+      :title="$t('Sequence Flow')"
+    />
+    <message-flow-button
+      @click="addMessageFlow()"
+      class="crown-config__icon"
+      :title="$t('Message Flow')"
+    />
+    <delete-button
```

Comment:
Might need to tweak this. I thought it was using the svg, but it seems to not work without the classes: 'svg-inline--fa fa-trash-alt fa-w-14'